examples/sensor-samplingrate/src/components/SensorItem.jsx

```diff
@@ -0,0 +1,74 @@
+import { Accordion, Button, Card, ListGroup } from "react-bootstrap";
+import React from "react";
+
+const _ = require("lodash");
+
+function listItem(item, index) {
+  return (
+    <ListGroup.Item
+      key={index}
+      className="border-0 px-0 py-2 text-capitalize bg-transparent"
+    >
+      <b className="pr-2 text-dark">{item.label}:</b>
+      {item.value}
+    </ListGroup.Item>
+  );
+}
+
+function SensorListItem(props) {
+  const status = _.get(props.sampling, "enable");
+  const sensor_items = [
+    {
+      label: "Sampling Status",
+      value:
+        status !== undefined
```

Comment:
I'd create a helper function instead of nesting if conditions

```
function samplingToString(sampling) {
 if ( .. ) {
  return "Enable"
} else if ( .. ) {
  return "Disable"
} else {
  return "Handled by device"
}
```

Reply:
I'll also add, i'm not a big fan of `undefined` checks. I'd prefer having a specific value to represent the state "handled by device"

Reply:
Done